action.yaml

```diff
@@ -26,16 +26,32 @@ runs:
     - name: Install Erlang for RabbitMQ
       run: |
         if [[ "$OS" == 'ubuntu-20.04' ]]; then
+          echo 'Installing erlang dependencies'
+          sudo apt-get update
+          sudo apt-get install -y gnupg apt-transport-https libsctp1
+          echo 'Clean older version of erlang-base'
+          sudo apt-get purge -yf erlang-base
           echo 'Installing erlang'
-          sudo scripts/rabbit_dependencies.sh debian focal
+          wget https://binaries2.erlang-solutions.com/ubuntu/pool/contrib/e/esl-erlang/esl-erlang_25.2-1~ubuntu~focal_amd64.deb
+          sudo dpkg -i esl-erlang_25.2-1~ubuntu~focal_amd64.deb
+        fi
+        if [[ "$OS" == 'ubuntu-22.04' ]]; then
+          echo 'Installing erlang dependencies'
+          sudo apt-get update
+          sudo apt-get install -y gnupg apt-transport-https libsctp1
+          echo 'Clean older version of erlang-base'
+          sudo apt-get purge -yf erlang-base
+          echo 'Installing erlang'
+          wget https://binaries2.erlang-solutions.com/ubuntu/pool/contrib/e/esl-erlang/esl-erlang_25.2-1~ubuntu~jammy_amd64.deb
+          sudo dpkg -i esl-erlang_25.2-1~ubuntu~jammy_amd64.deb
         fi
       shell: bash
       env:
         OS: ${{ inputs.os }}
 
     - name: Install dependencies including rmq
       run: |
-        if [[ ! -f "$FILE" ]] && [[ "$OS" == 'ubuntu-20.04' ]]; then
+        if [[ ! -f "$FILE" ]] && [[ "$OS" == 'ubuntu-20.04' || "$OS" == 'ubuntu-22.04' ]]; then
           echo 'Installing all volttron dependencies and RMQ'
           pip install wheel
           python bootstrap.py --all --rabbitmq --force
```

Comment:
Don't we have to have a separate --rabbitmq installation rather than all on one line.  I thought that was from the gitrunner?

@schandrika

Reply:
--all doesn't install rmq but --all + --rabbitmq installs everything including rabbit

Reply:
we are installing rmq only if it is ubuntu 20 or 22. other we install only zmq. This is how it was before.  I don't think our workflow need this check to be here because our test yml decide the os but useful if someone else is using this I suppose

Reply:
But in our testing you said that using --all --rabbitmq didn't work within the same command.  Which is why we broke it apart in the githlab runner.

Reply:
I am not sure why this happened in gitlab runner. But I tested now on my docker envs - Ubuntu 22 and Ubuntu 20  and --all with --rabbitmq install rmq in both ubuntu 22 and ubuntu 20 and both with and without activated enviornments